05-JWT-Basics/starter/middleware/error-handler.js

```diff
@@ -1,9 +1,13 @@
-const CustomAPIError = require('../errors/custom-error')
+const { CustomAPIError } = require("../errors/custom-error");
+const { statusCodes } = require("http-status-codes");
 const errorHandlerMiddleware = (err, req, res, next) => {
   if (err instanceof CustomAPIError) {
-    return res.status(err.statusCode).json({ msg: err.message })
+    return res.status(err.statusCode).json({ msg: err.message });
```

Comment:
Remember errors won’t necessarily have a status code now if they’re of the customAPIError class since we took that field out